### PR TITLE
Add dive tool

### LINKED ISSUE #513: Add `dive` tool for exploring each layer in a docker image
https://github.com/wagoodman/dive

Dive feels like the perfect tool to fit on the DevOps toolkit. It facilitates exploring a docker image, layer contents, and discovering ways to shrink the size of your Docker/OCI image.

## Are you a GitHub Sponsor (Yes/No?)

Check at https://github.com/sponsors/alexellis

- [ ] Yes
- [x] No

### PR DESCRIPTION
**Pull request by yankeexe**
Signed-off-by: Yankee Maharjan <yankee.exe@gmail.com>

Dive facilitates exploring a docker image, layer contents, and discovering ways to shrink the size of your Docker/OCI image.

Closes #513 

## Description

Add `dive` CLI to arkade.

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

```bash
go run . get dive 
```

## Are you a GitHub Sponsor (Yes/No?)

- [ ] Yes
- [x] No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by yankeexe** [comment]
@alexellis  @dirien Can you guys take a look at this? :slightly_smiling_face:

**Comment by dirien** [comment]
LGTM

**Comment by alexellis** [comment]
Merged, thank you for the suggestion

File: pkg/get/tools.go [hunk 1]
```diff
@@ -1609,5 +1609,32 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
...
+			{{- end -}}
+
+			{{$archiveStr := ""}}
+			{{- if HasPrefix .OS "ming" -}}
+			{{$archiveStr = ".zip"}}
+			{{- else -}}
+			{{$archiveStr = ".tar.gz"}}
+			{{- end -}}
+
+			https://github.com/{{.Owner}}/{{.Name}}/releases/download/v{{.Version}}/{{.Name}}_{{.Version}}_{{$osStr}}_amd64{{$archiveStr}}`,
```
Comment (alexellis) [inline pkg/get/tools.go:1635]: Could you try changing to BinaryTemplate and removing `https://github.com/{{.Owner}}/{{.Name}}/releases/download/` for a new feature I've added? And test removing the version.

**Pull request merged**